From patchwork Fri May 29 06:46:13 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 11577811
Return-Path: <SRS0=rOkw=7L=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id DEC611392
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 29 May 2020 06:46:20 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id C39EF2145D
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 29 May 2020 06:46:20 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (2048-bit key) header.d=canb.auug.org.au header.i=@canb.auug.org.au
 header.b="IJOIS0/Q"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1726167AbgE2GqS (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Fri, 29 May 2020 02:46:18 -0400
Received: from ozlabs.org ([203.11.71.1]:46139 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1725681AbgE2GqS (ORCPT <rfc822;kvm@vger.kernel.org>);
        Fri, 29 May 2020 02:46:18 -0400
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 49YFTM68PYz9sSp;
        Fri, 29 May 2020 16:46:15 +1000 (AEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1590734776;
        bh=l+Xvu2zy6hKZ6lhYjwD7Ye7643sbaA4BsCh6MX+Mf40=;
        h=Date:From:To:Cc:Subject:From;
        b=IJOIS0/Qce3klupjjUsZu7IruI9fPBWK5y71pBmirNG9rAJQmMgbg4Uh9ohw+i4iX
         bLK1egBbnQevdcHNSGnio/XHHuSxJEjpyNhTJ7XFWKuNiKDGQW/nJszJG8F3vduH7/
         44Dl0y+a17sN1CxqQAWeFnW8LB17zZe39l9MR8b3lcyjTIuPZ/nO17JYJF6IiqifD7
         j9hh4xV/mhiD2RxOiHq0YjWLievOWaIBepZM47EsRnSmT6FZ2xqb478YDbmBLfqWR1
         enZQziWeEWgq8OEfXko5eckwbMGOItE5YhuLymVw+o8+tN3NoO1cLwdr0vLf3U8Ppf
         6GTrVTZ7T6Uyw==
Date: Fri, 29 May 2020 16:46:13 +1000
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>, KVM <kvm@vger.kernel.org>,
        Heiko Carstens <heiko.carstens@de.ibm.com>,
        Vasily Gorbik <gor@linux.ibm.com>,
        Christian Borntraeger <borntraeger@de.ibm.com>
Cc: Linux Next Mailing List <linux-next@vger.kernel.org>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        David Hildenbrand <david@redhat.com>,
        Sven Schnelle <svens@linux.ibm.com>
Subject: linux-next: manual merge of the kvm tree with the s390 tree
Message-ID: <20200529164613.526f5865@canb.auug.org.au>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm tree got a conflict in:

  arch/s390/kvm/vsie.c

between commit:

  0b0ed657fe00 ("s390: remove critical section cleanup from entry.S")

from the s390 tree and commit:

  d075fc3154be ("KVM: s390: vsie: Move conditional reschedule")

from the kvm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc arch/s390/kvm/vsie.c
index 4fde24a1856e,ef05b4e167fb..000000000000
--- a/arch/s390/kvm/vsie.c
